resolwe/flow/management/commands/runexecutor_listener.py

```diff
@@ -0,0 +1,37 @@
+""".. Ignore pydocstyle D400.
+
+=================================
+Standalone Executor Contact Point
+=================================
+
+Command to run on local machine::
+
+    ./tests/manage.py runexecutor_listener
```

Comment:
Change it to `./manage.py` as Resolwe is mostly used as a library in other projects.